api_tests/dry/rfc003/test.js

```diff
@@ -23,8 +23,13 @@ const alpha_ledger_lock_duration = 144;
 
 const alice = test_lib.comit_conf("alice", {});
 const bob = test_lib.comit_conf("bob", {});
+const charlie = test_lib.comit_conf("charlie", {});
 
 const alice_final_address = "0x00a329c0648769a73afac7f9381e08fb43dbea72";
+const bob_comit_node_address =
+    bob.host + ":" + bob.config.comit.comit_listen.split(":")[1];
+const charlie_comit_node_address =
+    charlie.host + ":" + charlie.config.comit.comit_listen.split(":")[1];
```

Comment:
🔧 Make a method on charlie to get host and port from the config

Reply:
🔧 can't use just do that:
```suggestion
    charlie.config.comit.comit_listen;
```